complexipy/main.py

```diff
@@ -28,7 +26,7 @@
 root_dir = Path(__file__).resolve().parent.parent
 app = typer.Typer(name="complexipy")
 console = Console()
-version = "0.4.0"
+version = "0.4.0dev"
```

Comment:
I think that this could be the `0.5.0` version, you added an interesting feature